project-koku/koku
# [COST-2689] validate end date to be leq tomorrow

Fixes [COST-2689](https://issues.redhat.com/browse/COST-2689)

Changes proposed in this PR:
* remove `validate_start_date` and `validate_end_date` from `OrgQueryParamSerializer` and `TagsQueryParamSerializer` since these methods are inherited from `ParamSerializer`
* set validate end-date to tomorrow.

Testing Instructions:
1. Run `this command` ...
2. Ingest this data `with this other command` ...
3. Expect to see this result:
   ```
   GET http://localhost:8000/api/endpoint

   {json: response}
   ```
4.
5.

---
Additional Context:
[Corresponding smoke test updates](https://gitlab.cee.redhat.com/insights-qe/iqe-cost-management-plugin/-/merge_requests/1320)
